<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageAnnotator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="labelImagePath">
            <property name="text">
             <string>Images Path</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditImagePath"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnBrowseImagePath">
            <property name="text">
             <string>Browse Path</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelSelectedImage">
            <property name="text">
             <string>Selected Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGraphicsView" name="graphicsView">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="btnPreviousImage">
              <property name="text">
               <string>Previous Image</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnNextImage">
              <property name="text">
               <string>Next Image</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="labelCaption1">
            <property name="text">
             <string>Caption 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCaption1"/>
          </item>
          <item>
           <widget class="QLabel" name="labelCaption2">
            <property name="text">
             <string>Caption 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCaption2"/>
          </item>
          <item>
           <widget class="QLabel" name="labelCaption3">
            <property name="text">
             <string>Caption 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCaption3"/>
          </item>
          <item>
           <widget class="QLabel" name="labelCaption4">
            <property name="text">
             <string>Caption 4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCaption4"/>
          </item>
          <item>
           <widget class="QLabel" name="labelCaption5">
            <property name="text">
             <string>Caption 5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCaption5"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSaveCaptions">
            <property name="text">
             <string>Save Captions</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
